scanner/updater/import.go

```diff
@@ -0,0 +1,58 @@
+package updater
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/quay/claircore/pkg/ctxlock"
+	"github.com/stackrox/rox/scanner/datastore/postgres"
+	"github.com/stackrox/rox/scanner/matcher/updater/vuln"
+)
+
+// Load loads vulnerabilities into the Matcher DB.
+func Load(ctx context.Context, connString, vulnsURL string) error {
+	var success bool
+
+	pool, err := postgres.Connect(ctx, connString, "libvuln")
+	if err != nil {
+		return fmt.Errorf("connecting to postgres: %w", err)
+	}
+	defer func() {
```

Comment:
Why don't we always want to close the pool?

Reply:
Call me Mr. Copy-and-Paste. :smile: